azure-pipelines.yml

```diff
@@ -1,168 +1,8 @@
-# These jobs are triggered automatically and they test the code and the examples.
-
+# These jobs are triggered automatically and they test code, examples, and wheels.
+# Additional checks can be manually triggered
```

Comment:
AFAIK, this is more of a desideratum than a reality as soon as this is merged. Correct?

Reply:
Indeed )